lib/feedsme/index.js

```diff
@@ -308,7 +308,7 @@ Feedsme.prototype._triggerStrategy = function _triggerStrategy({ releaseLine, ro
   if (!releaseLine) return { strategy: 'legacy', trigger: true };
 
   const { name, version: rootVersion } = rootPkg;
-  const { version, dependencies } = pkg;
+  const { dependencies } = pkg;
```

Comment:
`version` was unused here, maybe part of the `undefined` stuff we were seeing, should it have actually been used?

Reply:
@msluther might have been leftover from when I thought magically bunching major versions was a good idea. I think its unrelated